src/interfaces/IFractionalGovernor.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
We should have an issue to review license identifiers, not sure if this is correct but also don't want to block on it

Reply:
Tracking in https://github.com/ScopeLift/flexible-voting/issues/28